javascript/code-challenges/graph/README.md

```diff
@@ -0,0 +1,31 @@
+# Binary Trees
+
+## Challenge 15
+
+Authors: Tahmina Ringer, Ashley Moore and Garrett Citron
+
+- Implemented a Graph structure that is represented as an adjacency list, and includes methods `addNode()`, `addEdge()`, `getNodes()`, `getNeighbors()` and `size()`. 
+
+## Approach & Efficiency
+
+I wrote my code first and then wrote tests to validate the code.
+
+## Big O
+
+worst case - O(n)
+Space - O(n)
+
+## Testing
+
+Write tests to prove the following functionality:
+
+### Challenge 35 testing
+
+- Node can be successfully added to the graph
+- An edge can be successfully added to the graph
+- A collection of all nodes can be properly retrieved from the graph
+- All appropriate neighbors can be retrieved from the graph
+- Neighbors are returned with the weight between nodes included
+- The proper size is returned, representing the number of nodes in the graph
+- A graph with only one node and edge can be properly returned
+- An empty graph properly returns null
```

Comment:
Once you have the two tests passing that are skipped and the one that's failing passing I will regrade for you!